apps/src/code-studio/components/progress/MiniViewTopRow.story.jsx

```diff
@@ -7,6 +7,12 @@ import progress from '@cdo/apps/code-studio/progressRedux';
 export default storybook => {
   const initialState = {
     progress: {
+      stages: [
+        {
+          levels: []
+        }
+      ],
+      focusAreaStageIds: [],
```

Comment:
I was also fooled. Though this particular component doesn't use this state, descendants do extract it from the store via connect.